apps/hash-frontend/src/blocks/page/create-suggester/create-suggester.tsx

```diff
@@ -319,7 +319,7 @@ export const createSuggester = (
             case "@":
               jsx = (
                 <MentionSuggester
-                  search={search.substring(1)}
+                  search={search}
```

Comment:
The documentation on `search` suggests:
```ts
  /** matched search string including its leading trigger-char */
  search: string;
```
so the `MentionSuggester` and `BlockSuggester` would search for `@user` or `/block` instead of `user` or `block`. Either this change is wrong or the documentation is outdated.

Reply:
I guess it's outdated, updated the documentation here https://github.com/hashintel/hash/pull/2586/commits/ba0e858c728310997e42e2c0b964fb9d919a3207
<img width="315" alt="image" src="https://github.com/hashintel/hash/assets/39553853/e7aeb456-0b10-48b1-b37c-5c9ff12c863f">